Modules/Microsoft365DSC/DSCResources/MSFT_TeamsCallQueue/MSFT_TeamsCallQueue.schema.mof

```diff
@@ -3,11 +3,21 @@ class MSFT_TeamsCallQueue : OMI_BaseResource
 {
     [Key, Description("The Name parameter specifies a unique name for the Call Queue.")] String Name;
     [Write, Description("The Name parameter specifies a unique name for the Call Queue.")] UInt16 AgentAlertTime;
+
     [Write, Description("The AllowOptOut parameter indicates whether or not agents can opt in or opt out from taking calls from a Call Queue.")] Boolean AllowOptOut;
     [Write, Description("The DistributionLists parameter lets you add all the members of the distribution lists to the Call Queue. This is a list of distribution list GUIDs. A service wide configurable maximum number of DLs per Call Queue are allowed. Only the first N (service wide configurable) agents from all distribution lists combined are considered for accepting the call. Nested DLs are supported. O365 Groups can also be used to add members to the Call Queue.")] String DistributionLists[];
     [Write, Description("The UseDefaultMusicOnHold parameter indicates that this Call Queue uses the default music on hold. This parameter cannot be specified together with MusicOnHoldAudioFileId.")] Boolean UseDefaultMusicOnHold;
     [Write, Description("The WelcomeMusicAudioFileId parameter represents the audio file to play when callers are connected with the Call Queue. This is the unique identifier of the audio file.")] String WelcomeMusicAudioFileId;
+    [Write, Description("The MusicOnHoldFileName parameter represents the name of the audio file to play when callers are placed on hold within the Call Queue.")] String MusicOnHoldFileName;
     [Write, Description("The MusicOnHoldFileContent parameter represents music to play when callers are placed on hold. This is the unique identifier of the audio file. This parameter is required if the UseDefaultMusicOnHold parameter is not specified.")] String MusicOnHoldAudioFileId;
+    [Write, Description("The NoAgentAction parameter designates the action toto take if the no agents condition is reached. The NoAgentAction property must be set to one of the following values: Queue, Disconnect, Forward, Voicemail, and SharedVoicemail. The default value is Queue."), ValueMap{"Queue","Forward","Disconnect","Forward","Voicemail","SharedVoicemail"},Values{"Queue","Forward","Disconnect","Forward","Voicemail","SharedVoicemail"}] String NoAgentAction;
```

Comment:
In the PR title I see that the 2nd Forward probably should be combined with the Voicemail string that is immediately behind it, so ForwardVoicemail instead of Forward, Voicemail.